modules/modelops/pages/feast-teradata.adoc

```diff
@@ -0,0 +1,253 @@
+= Using Teradata with FEAST
+:experimental:
+:page-author: Mohammmad Taha Wahab , Mohammad Harris Mansur and Dr. Will Fleury
+:page-email: mohammadtaha.wahab@teradata.com , mohammadharris.mansur@teradata.com and will.fleury@teradata.com
+:page-revdate: January 5th, 2023
+:description: Feast Enterprise Feature Store Connector with Teradata
+:keywords: data warehouses, analytics, teradata, vantage, time series, business intelligence, enterprise analytics, feature store, Feast, connector
+
+== Introduction
+
+Feast's connector for Teradata is a complete implementation with support for all features and uses teradata as an online and offline store.
+
+== Prerequisites
+
+You need access to a Teradata Vantage instance.
+
+include::ROOT:partial$vantage.express.options.adoc[]
+
+
+== Overview
+We recommend you familiarize yourself with the terminology and concepts of feast by reading the official https://docs.feast.dev/[FEAST documentation]
+
+This document demonstrates how developers can integrate `Teradata's offline and online store` for Feast. Teradata's offline stores allow users to use any underlying data store as their offline feature store. Features can be retrieved from the offline store for model training, and can be materialized into the online feature store for use during model inference.
```

Comment:
The text contains several grammar mistakes, example below. Could you please check the doc with Grammarly?
```suggestion
This document demonstrates how developers can integrate `Teradata's offline and online store` with Feast. Teradata's offline stores allow users to use any underlying data store as their offline feature store. Features can be retrieved from the offline store for model training, and can be materialized into the online feature store for use during model inference.
```